Corona-Warn-App/build.gradle

```diff
@@ -162,6 +161,7 @@ android {
         variant.buildConfigField "String", "ENVIRONMENT_TYPE_DEFAULT", "\"${flavor.envTypeDefault[typeName]}\""
 
         if (flavor.name == "deviceForTesters") {
+            buildConfigField "String", "GIT_COMMIT_SHORT_HASH", "\"${getHash()}\""
```

Comment:
What was the specific error message?

If this just should be set for specific versions, it should probably be `variant.buildConfigField`?

Reply:
Internal pipeline: A problem occurred starting process 'command 'git''

Reply:
prefixed variant.